cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java

```diff
@@ -194,6 +194,16 @@ public void onError(ConsentLibException error) {
             public void onAction(ConsentAction action) {
                 GDPRConsentLib.this.onAction(action);
             }
+
+            @Override
+            public void onBackPressAction() {
+                if (webView.canGoBack()){
```

Comment:
here you should change `webview` by `this`

Reply:
you don't even need to call this.goBack() here. Just call `GDPRConsentLib.this.onAction(ConsentAction.getEmptyDismissAction(isPmOn));`
and let the magic happen :)

Reply:
will  do that.